GliaWidgets/Component/CallButtonBar/Button/CallButtonStyle.swift

```diff
@@ -160,4 +164,64 @@ public struct CallButtonStyle {
             }
         }
     }
+
+    private mutating func applySelectedBarButtonConfiguration(button: RemoteConfiguration.BarButtonStates?) {
+        button?.selected?.background?.type.map { backgroundType in
+            switch backgroundType {
+            case .fill:
+                button?.selected?.background?.value.map {
+                    selected.backgroundColor = UIColor(hex: $0[0])
+                }
+            case .gradient:
+
```

Comment:
NIT: I don't think that the comment needs a space above and below. Also it could be aligned to the indentation that the actual code is going to have rather than the indentation of the case statement. So, like

```
case .gradient:
    // The logic for gradient has not been implemented
    break
```

In case you decide to take the suggestion, then it would apply to all comments. Otherwise leave all as they are right now, as long as they are consistent with each other.

Reply:
This comment will be removed in a gradient PR, I just left it as were the other ones so that it would look roughly the same. Once This PR is merged, I will take care of this issue there :)